htmd/parameterization/dihedral.py

```diff
@@ -71,53 +72,15 @@ def __init__(self):
         self._all_target_energies = None
         self._angle_values_rad = None
 
+        self.parmedMode = False
+        self._prm = None
+        self._rotatable_dihedrals = None
```

Comment:
Call the parameterizable dihedral detector internally, so no need to pass this from outside.

Reply:
Withe the current design of parameterize I don't agree to this change, as it leads to calculation duplication. We will need to discuss the class design and if we end up making the classes stand-alone I agree to remove it.

Reply:
But for a following PR. Not this one